client/src/components/MediaRenderer.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+
+const getFileTypeFromUrl = (url) => {
+  const extension = url.split('.').pop().toLowerCase();
+  if (extension === 'mp4' || extension === 'mov') {
+    return 'video';
+  } else if (extension === 'jpg' || extension === 'jpeg' || extension === 'png' || extension === 'gif') {
+    return 'image';
+  } else {
+    return 'unknown';
+  }
+};
+
+const VideoRenderer = ({ fileUrl, numberOfMedia, fileType }) => (
```

Comment:
Video & Image Renderer components could probably be taken out of this file for separation of usages :P